README.md

```diff
@@ -32,12 +32,23 @@ To use rules provided by the plugin, use the following:
 }
 ```
 
+Some rules are fixable with `eslint --fix`. For example the [place-fields](docs/rules/place-fields.md) rule.
+
+```js
+service.getDetails({place_id: 'foo'})
+```
+becomes
+
+```js
+service.getDetails({fields: /** TODO: Add necessary fields to the request */ [], place_id: 'foo'})
+```
+
 ## Rules
 
 | Rule                                       | Description                | Configurations   | Type      |
 | ------------------------------------------ | -------------------------- | ---------------- | ------------ |
 | [no-api-keys](docs/rules/no-api-keys.md)   | Keep API keys out of code. | ![recommended][] | ![suggest][] |
-| [place-fields](docs/rules/place-fields.md) | Always use place fields.   | ![recommended][] | ![suggest][] |
+| [place-fields](docs/rules/place-fields.md) | Always use place fields.   | ![recommended][] | ![fixable][] |
```

Comment:
Let's add this shortlink to guidance about Place Fields. The /docs/rules/place-fields.md description seems to be focused on detecting Place Details and Place Autocomplete requests that are missing fields. Could we split the rule into a Details fields rule and an Autocomplete fields rule? 

- For Place Details, the resolution guidance is at: https://goo.gle/3H0TxxG so it would be helpful to add this to the rule description.
- For Place Autocomplete, the resolution guidance is at: https://goo.gle/3sp2XyS

Note that [Find Place requests](https://developers.google.com/maps/documentation/javascript/places#find_place_requests) also support field masks in order to manage Data SKU costs. However, since `fields` is a required field for Find Place requests, that is less likely to need lint checks to catch requests that are missing the fields.

Reply:
Updated the description of the rule with the links in the latest commit. Opened a new issue for splitting #31.